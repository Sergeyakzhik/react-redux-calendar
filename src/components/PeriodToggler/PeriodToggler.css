.btn-group {
  margin-top: 0.3rem;
}

.btn-group button {
  color: black;
  background-color: white;
  width: 3rem;
  height: 2.7rem;
}

.btn:hover {
  background-color: #a0e070;
}

.btn-secondary.focus, .btn-secondary:focus {
  box-shadow: none;
}

.btn-secondary:not(:disabled):not(.disabled).active, .btn-secondary:not(:disabled):not(.disabled):active, .show>.btn-secondary.dropdown-toggle {
  color: #black;
  background-color: #a0e070;
  border-color: transparent;
}

i {
  pointer-events: none;
}
