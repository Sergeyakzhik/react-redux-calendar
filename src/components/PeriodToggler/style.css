.btn-group {
  margin-top: 0.3rem;
}

.style1 .btn-group button {
  color: #ffffff;
  background-color: #181818;
  width: 3rem;
  height: 2.7rem;
  border-color: black;
}

.style1 .btn:hover {
  background-color: #4e4e4e;
}

.style1 .btn-secondary.focus, .btn-secondary:focus {
  box-shadow: none;
}

.style1 .btn-secondary:not(:disabled):not(.disabled).active, .style1 .btn-secondary:not(:disabled):not(.disabled):active, .show>.btn-secondary.dropdown-toggle {
  color: black;
  background-color: #757575;
  border-color: transparent;
}

.style2 .btn-group button {
  color: black;
  background-color: #daf6da;
  width: 3rem;
  height: 2.7rem;
}

.style2 .btn:hover {
  background-color: #eafbc4;
}

.style2 .btn-secondary.focus, .btn-secondary:focus {
  box-shadow: none;
}

.style2 .btn-secondary:not(:disabled):not(.disabled).active, .style2 .btn-secondary:not(:disabled):not(.disabled):active, .show>.btn-secondary.dropdown-toggle {
  color: black;
  background-color: #aeffb0;
  border-color: transparent;
}

i {
  pointer-events: none;
}
