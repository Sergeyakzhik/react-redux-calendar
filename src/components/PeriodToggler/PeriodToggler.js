import React from 'react';
import './PeriodToggler.css';

let PeriodToggler = props => {
  return (
    <div className="btn-group" role="group">
      <button onClick={props.onPeriodTogglerClick} type="button" className="btn btn-secondary left-button" id="left-button"><i className="fas fa-chevron-left"></i></button>
      <button onClick={props.onPeriodTogglerClick} type="button" className="btn btn-secondary right-button" id="right-button"><i className="fas fa-chevron-right"></i></button>
    </div>
  );
}

export default PeriodToggler;
