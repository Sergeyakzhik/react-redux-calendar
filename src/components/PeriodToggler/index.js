import React from 'react';
import './style.css';

const PeriodToggler = props => (
  <div className="btn-group" role="group">
    <button
      onClick={props.onPeriodTogglerClick}
      type="button"
      className="btn btn-secondary left-button"
      id="left-button"
    >
      <i className="fas fa-chevron-left" />
    </button>
    <button
      onClick={props.onPeriodTogglerClick}
      type="button"
      className="btn btn-secondary right-button"
      id="right-button"
    >
      <i className="fas fa-chevron-right" />
    </button>
  </div>
);

export default PeriodToggler;
