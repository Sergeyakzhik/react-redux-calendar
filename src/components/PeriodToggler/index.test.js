import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'

import PeriodToggler from './index';

Enzyme.configure({ adapter: new Adapter() })

describe('PeriodToggler', () => {
  let props = {
    onPeriodTogglerClick: () => {}
  }

  let component = shallow(<PeriodToggler {...props} />);

  it('renders PeriodToggler component', () => {
    expect(component).toMatchSnapshot();
  })
})
