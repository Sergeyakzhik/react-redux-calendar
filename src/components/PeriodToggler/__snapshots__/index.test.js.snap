// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PeriodToggler render PeriodToggler component 1`] = `
<div
  className="btn-group"
  role="group"
>
  <button
    className="btn btn-secondary left-button"
    id="left-button"
    onClick={[Function]}
    type="button"
  >
    <i
      className="fas fa-chevron-left"
    />
  </button>
  <button
    className="btn btn-secondary right-button"
    id="right-button"
    onClick={[Function]}
    type="button"
  >
    <i
      className="fas fa-chevron-right"
    />
  </button>
</div>
`;
